solarforecastarbiter/metrics/errors.py

```diff
@@ -0,0 +1,18 @@
+"""
+Metric module specific errors.
+"""
+
+
+class SfaMetricsError(Exception):
```

Comment:
I don't think we've defined any special SFA errors. Not opposed if it makes things easier, but not required either.

Reply:
Okay. Let me know if you want me to remove them for now.